ipaplatform/redhat/tasks.py

```diff
@@ -518,9 +514,9 @@ def _create_tmpfiles_dir(self, name, mode, uid, gid):
         os.chmod(name, mode)
         os.chown(name, uid, gid)
 
-    def create_tmpfiles_dirs(self):
+    def create_tmpfiles_dirs(self, ipaapi_user):
```

Comment:
Hi @tiran, thanks for reviewing this.
That sounds a good idea to me, but in some places only `create_tmpfiles_dirs` is called, like:
* https://github.com/freeipa/freeipa/blob/master/ipaserver/install/server/install.py#L1055
* https://github.com/freeipa/freeipa/blob/master/ipaserver/install/server/replicainstall.py#L1359

Which leads to: Is harmful to change this calls to `configure_tmpfiles` (after adding `systemd-tmpfiles` to the method)?